mne/label.py

```diff
@@ -1659,6 +1660,159 @@ def _grow_nonoverlapping_labels(subject, seeds_, extents_, hemis, vertices_,
     return labels
 
 
+def random_parcellation(subject, n_parcel, hemis, subjects_dir=None,
+                        surface='white'):
```

Comment:
you should use np.random and have a random_state parameter to make the result reproducible.

git grep random_state

to see how we do it.